ReleaseNotes.md

```diff
@@ -25,6 +25,7 @@ See [http://git-scm.com/](http://git-scm.com/) for further details about Git inc
 * Older versions of the Windows Explorer do *not* calculate Git for Windows' on-disk size correctly, as it is unaware of hard links. Therefore, it might look like Git for Windows takes up 1.5GB when in reality it is about a third of that.
 * Git command hints are designed for a POSIX shell, this can lead to issues when using them **as is** in non-POSIX shells like PowerShell, [such as this ticket](https://github.com/git-for-windows/git/issues/2785).
 * When pushing via the `git://` protocol, Git for Windows may hang indefinitely. The last console output in this case is typically `Writing objects: 100%`. Until issue [#907](https://github.com/git-for-windows/git/issues/907) is addressed, run this command once as a work-around: `git config sendpack.sideband false`.
+* If system-wide Mandatory ASLR is enabled some Git for Windows binaries may fail to work. This is a [related issue](https://github.com/git-for-windows/git/issues/1412).
```

Comment:
I re-read that ticket, and I don't think that it will be helpful to people who run into Mandatory ASLR problems. Maybe a better idea would be to summarize the finding that some (or all) `.exe` files in `/usr/bin/` need to be processed via `Set-ProcessMitigation`.